packages/functional-tests/pages/settings/recoveryKey.ts

```diff
@@ -12,10 +12,23 @@ export class RecoveryKeyPage extends SettingsLayout {
     return this.page.innerText('[data-testid=datablock] span');
   }
 
+  async invalidRecoveryKeyError() {
+    return this.page.innerText('#error-tooltip-159');
```

Comment:
Q: what's the specific `id`?
Does this change w/ l10n changes?

Reply:
Not sure. But I have seen 2 different error codes so far. One was for Sign in error tooltip and other is this one.